connexion/apps/flask_app.py

```diff
@@ -148,13 +149,17 @@ def run(self,
             wsgi_container = tornado.wsgi.WSGIContainer(self.app)
             http_server = tornado.httpserver.HTTPServer(wsgi_container, **options)
             http_server.listen(self.port, address=self.host)
+            if self.debug:
+                tornado.autoreload.start()
             logger.info('Listening on %s:%s..', self.host, self.port)
             tornado.ioloop.IOLoop.instance().start()
         elif self.server == 'gevent':
             try:
                 import gevent.pywsgi
             except ImportError:
                 raise Exception('gevent library not installed')
+            if self.debug:
+                logger.warning("gevent server doesn't support debug mode. Please switch to flask/tornado server.")
```

Comment:
Haven't tested this myself yet, but is it possible to enable debug mode by setting it on `self.app` that is getting passed to the gevent server?

```
self.app.debug = True
```

Reply:
Unfortunately, this didn't work. `Gevent` module doesn't handle `self.app.debug` flag 😞

Reply:
Alright, thanks for checking!